src/github/pullRequestManager.ts

```diff
@@ -844,6 +844,44 @@ export class PullRequestManager {
 		return parseGraphQLComment(comment);
 	}
 
+	async addCommentReaction(pullRequest: PullRequestModel, graphNodeId: string, reaction: vscode.CommentReaction): Promise<AddReactionResponse> {
+		let reactionEmojiToContent = getReactionGroup().reduce((prev, curr) => {
+			prev[curr.label] = curr.title;
+			return prev;
+		}, {} as { [key: string]: string });
+		const { mutate } = await pullRequest.githubRepository.ensure();
+		const { data } = await mutate<AddReactionResponse>({

```

Comment:
Do we intend to also support reactions for GHE?

Reply:
We don't. This API won't be called when there is no graphql support.